kotlinpoet/src/test/java/com/squareup/kotlinpoet/KmSpecsTest.kt

```diff
@@ -62,24 +69,18 @@ class KmSpecsTest {
   fun properties() {
     val typeSpec = Properties::class.toTypeSpec()
 
-    assertThat(typeSpec.propertySpecs).hasSize(4)
-
-    val fooProp = typeSpec.propertySpecs.find { it.name == "foo" } ?: throw AssertionError(
-        "Missing foo property")
-    assertThat(fooProp.type).isEqualTo(String::class.asClassName())
-    assertThat(fooProp.mutable).isFalse()
-    val barProp = typeSpec.propertySpecs.find { it.name == "bar" } ?: throw AssertionError(
-        "Missing bar property")
-    assertThat(barProp.type).isEqualTo(String::class.asClassName().copy(nullable = true))
-    assertThat(barProp.mutable).isFalse()
-    val bazProp = typeSpec.propertySpecs.find { it.name == "baz" } ?: throw AssertionError(
-        "Missing baz property")
-    assertThat(bazProp.type).isEqualTo(Int::class.asClassName())
-    assertThat(bazProp.mutable).isTrue()
-    val listProp = typeSpec.propertySpecs.find { it.name == "aList" } ?: throw AssertionError(
-        "Missing baz property")
-    assertThat(listProp.type).isEqualTo(List::class.parameterizedBy(Int::class))
-    assertThat(listProp.mutable).isTrue()
+    //language=kotlin
+    assertThat(typeSpec.trimmedToString()).isEqualTo("""
+      class Properties {
+        var aList: kotlin.collections.List<kotlin.Int> = TODO("Stub!")
+
+        val bar: kotlin.String? = null
```

Comment:
Should nullable fields also get assigned a `TODO` instead of null?

Reply:
I don't think so, unless you think that'd make it more clear that it's a placeholder